modules/table/src/test/java/org/apache/ignite/internal/table/distributed/replication/PartitionReplicaListenerTest.java

```diff
@@ -210,7 +213,7 @@
 import org.mockito.quality.Strictness;
 
 /** Tests for partition replica listener. */
-@ExtendWith(MockitoExtension.class)
+@ExtendWith({MockitoExtension.class, ConfigurationExtension.class})
```

Comment:
In this PR I see both repeated annotations and explicit arrays. It would be nice to only see one of two. Repeated annotations, for example.